scripts/index.js

```diff
@@ -1,7 +1,136 @@
 }
+workContain = `<section class="title">
+                      <h3>My Recent Works</h3>
+                      <p class="s"></p>
+              </section>
+            <section class="works grid-container">
+              ${workContain} 
+              ${workContain}
+            </section>`;
+const workSection = document.getElementById('test');
+workSection.innerHTML = `${workContain}`;
+
+// ============================================================================================
+// Actions on btn see details popup
+
+function openPopUp() {
+  console.log('load modal pop up window');
+  const pop = `<section class="container">
+                    <div class="grid-item pop_head">
+                      <nav class="menu">
+                        <img src="images/Normal Button/Tertiary/Enabled.svg" alt="" id="close_pop">
+                      </nav>
+                    </div>
+                    <div class="grid-item pop_title">
+                      <span class="title">
+                       Keeping track of hundreds of components
+                      </span>
+                      <ul class="tag_ctn">
+                        <li class="tag"><span class="tag_txt">Ruby On Rails</span></li>
+                        <li class="tag"><span class="tag_txt">CSS</span></li>
+                        <li class="tag"><span class="tag_txt">Javascript</span></li>
+                      </ul>
+                    </div>
+                    <div class="grid-item pop_details">
+                      <div class="details">
+                      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.
+                      <br><br>
+                      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.
+                      </div>
+                    </div>
+                    <div class="grid-item pop_links">
+                      <div class="links action_2 centered">
+                        <button class="see">
+                          <span class="see_txt">See Live</span>
+                          <img src="images/Icons/see live icon.svg">
+                        </button>
+                      </div>
+                      <div class="links action_2 centered">
+                        <button class="see">
+                          <span class="see_txt">See Source</span>
+                          <img src="images/Icons/Vector.svg">
+                        </button>
+                      </div>
+                    </div>
+
+                </section>`;
+  popUpWindows.innerHTML = `${pop}`;
+  popUpWindows.className = 'pop_up';
 
-if (IsPositive(10)) {
-  // console.log('test passed');
```

Comment:
I suggest you remove all commented code in your file to make your code base look more professional and follow best practices. 😄